web/app/models.py

```diff
@@ -174,6 +174,8 @@ def to_dict(self):
 KOMI_VALUES = ['0', '0.5', '1', '1.5', '2', '2.5', '3.0', '3.5', '4', '4.5',
     '5', '5.5', '6', '6.5', '7', '7.5', '8', '8.5', '9', '9.5']
 TIE_BREAKS = ['SOS', 'SODOS', 'Face to Face Result', 'Random Procedure']
+MATCH_RESULTS = ['White Wins', 'Black Wins', 'Jigo', 'No Result',
```

Comment:
My guess is that the game outcomes weren't intended as a literal enum list, but rather as a warning that your data format should accommodate these types of outcomes.

A double forfeit happens if both players are no-show. I've had some of these for the Collegiate Go League.